### PR TITLE
V3.0.2

### PR DESCRIPTION
**Pull request by nefischer**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    environment:
+      docker_username: ukhomeofficedigital+helm
+    secrets:
+      - docker_password
+    commands:
+      - docker login -u="${DOCKER_USERNAME}" -p=$${DOCKER_PASSWORD} ${DOCKER_REGISTRY}
+      - docker tag ${DOCKER_REGISTRY}/${DOCKER_REPO}:$${DRONE_COMMIT_SHA} ${DOCKER_REGISTRY}/${DOCKER_REPO}:latest
+      - docker tag ${DOCKER_REGISTRY}/${DOCKER_REPO}:$${DRONE_COMMIT_SHA} ${DOCKER_REGISTRY}/${DOCKER_REPO}:$${DRONE_TAG}
+      - docker push ${DOCKER_REGISTRY}/${DOCKER_REPO}:$${DRONE_COMMIT_SHA}
+      - docker push ${DOCKER_REGISTRY}/${DOCKER_REPO}:latest
```
Comment (KashifSaadat) [inline .drone.yml]: We may not want to overwrite latest on every tag push

**Review by KashifSaadat — APPROVED** [review]

File: .drone.yml (hunk 1 above)
Comment (nefischer) [reply .drone.yml]: Good call. I'll add a pipeline step triggered by a push to that master to deal with it. That should cover it, shouldn't it?

Comment (KashifSaadat) [reply .drone.yml]: Yup that would be good, thank you :)

**Review by KashifSaadat — APPROVED** [review]
LG, cheers for the update. 👍

**Pull request merged**